src/BuildIntegration/findvcvarsall.bat

```diff
@@ -15,10 +15,16 @@ IF "%vsBase%"=="" GOTO :ERROR
 CALL "%vsBase%\vc\Auxiliary\Build\vcvarsall.bat" %1% > NUL
 
 FOR /F "delims=" %%W IN ('where link') DO (
-    FOR %%A IN ("%%W") DO ECHO %%~dpA
-    EXIT /B 0
+    FOR %%A IN ("%%W") DO ECHO %%~dpA#
+    goto :capture_lib_paths
 )
 
+:capture_lib_paths
+ECHO %LIB%#
+ECHO %LIBPATH%
```

Comment:
I do not think we need LIBPATH here - it should not matter to the linker. See https://stackoverflow.com/questions/20483619/what-is-the-difference-between-the-lib-and-libpath-environment-variables-for-ms

Reply:
Thanks, I have removed LIBPATH.

This bit is interesting and confused me as well:

> And yet /LIBPATH is an option for the linker command line that overrides the environment. Aha!... I see it now. Doc indicates that /LIBPATH overrides the LIB environment variable, not the LIBPATH environment variable. It's just a (bad) coincidence of names.